spec/features/managing_step_by_step_pages_spec.rb

```diff
@@ -163,6 +163,18 @@
       then_I_should_see "Scheduled to be published at"
       and_there_should_be_no_schedule_button
     end
+
+    scenario "User tries using invalid values for schedule date and time" do
```

Comment:
It's great that you've added this test, but it's not obvious which bits of commit 0819de3deb733d6f2dbaa8223c71db72b0a7aedb these tests relate to.

Can you go back an split 0819de3deb733d6f2dbaa8223c71db72b0a7aedb into one commit that adds the global error, along with the appropriate lines from this test, and another that adds in the inline error, also with the appropriate lines from this test.

Reply:
Done 👍